src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2020,15 +2020,14 @@ bool processOnChangeConditionForSubscription
 *
 */
 BSONArray processConditionVector
-(
-  NotifyConditionVector*           ncvP,
+(NotifyConditionVector*           ncvP,
   EntityIdVector                   enV,
   AttributeList                    attrL,
-  std::string                      subId,
```

Comment:
EntityIdVector + AttributeList ...  could be sent as refs also, right?
And BSONArray as a return value ... also an object